torch/_inductor/freezing.py

```diff
@@ -222,7 +222,12 @@ def freeze(
     aot_autograd_gm.recompile()
 
     from torch._inductor.compile_fx import fake_tensor_prop
+    from torch._inductor.fx_passes.post_grad import view_to_reshape
```

Comment:
this can be imported at the top of the file

Reply:
changed.